mender-dist-packages-building/install-armv6-cross-toolchain

```diff
@@ -0,0 +1,170 @@
+#! /bin/bash
+
+set -ex
+
+# To provide support for Raspberry Pi Zero W a toolchain tuned for ARMv6 architecture must be used.
+# https://northerntech.atlassian.net/browse/MEN-2399
+ARMV6_TOOLCHAIN_ROOT="${ARMV6_TOOLCHAIN_ROOT:?No toolchain set}"
+TOOLCHAIN_PREFIX=${ARMV6_TOOLCHAIN_ROOT%/*}
+RASPBIAN_VERSION="${VERSION:?No Raspbian version set}"
+DEBIAN_URL="http://raspbian.raspberrypi.org/raspbian/dists/${RASPBIAN_VERSION}/main/binary-armhf/Packages.gz"
+BOOTLIN_URL="https://toolchains.bootlin.com/downloads/releases/toolchains/armv6-eabihf/tarballs/"
+ARMV6_TOOLCHAIN="${ARMV6_TOOLCHAIN_ROOT#/opt/}"
+
+declare -a required_packages=(
+    liblzma5
+    liblzma-dev
+    libssl1.1
+    libssl-dev
+    libglib2.0-0
+    libglib2.0-dev
+    libpcre3
+    libpcre3-dev
+    zlib1g
+    zlib1g-dev
+    libffi-dev
+    libmount1
+    libmount-dev
+    libselinux1
+    libselinux1-dev
+    libpcre2-8-0
+    libpcre2-dev
+    libblkid1
+    libblkid-dev
+    libuuid1
+    uuid-runtime
+    uuid-dev
+    liblmdb-dev
+    liblmdb0
+    libacl1
+    libarchive-dev
+    libarchive13
+    libasan5
+    libatomic1
+    libattr1
+    libblkid1
+    libboost-atomic-dev
+    libboost-atomic1.67-dev
+    libboost-atomic1.67.0
+    libboost-chrono-dev
+    libboost-chrono1.67-dev
+    libboost-chrono1.67.0
+    libboost-date-time1.67-dev
+    libboost-date-time1.67.0
+    libboost-dev
+    libboost-filesystem-dev
+    libboost-filesystem1.67-dev
+    libboost-filesystem1.67.0
+    libboost-log-dev
+    libboost-log1.67-dev
+    libboost-log1.67.0
+    libboost-regex-dev
+    libboost-regex1.67-dev
+    libboost-regex1.67.0
+    libboost-serialization1.67-dev
+    libboost-serialization1.67.0
+    libboost-system1.67-dev
+    libboost-system1.67.0
+    libboost-thread-dev
+    libboost-thread1.67-dev
+    libboost-thread1.67.0
+    libboost1.67-dev
+    libbz2-1.0
+    libgcc1
+    libgomp1
+    libicu-dev
+    libicu63
+    libidn2-0
+    liblmdb-dev
+    liblmdb0
+    liblz4-1
+    liblzma5
+    libmount1
+    libnettle6
+    libpcre3
+    libselinux1
+    libssl-dev
+    libssl1.1
+    libxml2
+    libzstd1
+)
```

Comment:
I cannot possibly review the list. I trust you here. (Having too many packages is harmless, and having too few would make the build fail...)

Reply:
Agreed